pynotify/models.py

```diff
@@ -41,6 +41,8 @@ class BaseTemplate(BaseModel):
     text = models.TextField(null=True, blank=True, verbose_name=_l('text'))
     trigger_action = models.CharField(max_length=2500, null=True, blank=True,
                                       verbose_name=_l('trigger action'))
+    trigger_action_alternative = models.CharField(max_length=2500, null=True, blank=True,
+                                                  verbose_name=_l('trigger action alternative'))
```

Comment:
You forgot to generate translations, see https://django-pynotify.readthedocs.io/en/latest/contributing.html#get-started on how to do that.

Reply:
now it's there

Reply:
why is used name alternative?

Reply:
@rubickcz you have no problem with name of the field? Make it sense with library naming confention. Maybe use array field with more trigger actions or custom model? Just asking